### ISSUE TITLE
Broken querystring after qs.stringify

### ISSUE THREAD

**Issue by kof** [visionmedia/superagent#411]
I have the following url

http://git.ozlabs.org/?p=yaboot.git;a=rss;f=second/file.c;opt=--no-merges

parsed output of it is

  { protocol: 'http:',
    slashes: true,
    auth: null,
    host: 'git.ozlabs.org',
    port: null,
    hostname: 'git.ozlabs.org',
    hash: null,
    search: '?p=yaboot.git;a=rss;f=second/file.c;opt=--no-merges',
    query: { p: 'yaboot.git;a=rss;f=second/file.c;opt=--no-merges' },
    pathname: '/',
    path: '/?p=yaboot.git;a=rss;f=second/file.c;opt=--no-merges',
    href: 'http://git.ozlabs.org/?p=yaboot.git;a=rss;f=second/file.c;opt=--no-merges' }

**Comment by defunctzombie** [comment]
Closing this for no response. If this is still broken (sounds likely) can we make a simple explanation of the issue and failing test. The current issue is hard to follow :)
